CIP-XXXX/README.md

```diff
@@ -0,0 +1,60 @@
+---
+CIP: ?
+Title: CIP-30 ext: Bulk transaction signing
+Category: Wallets
+Status: Proposed
+Authors:
+    - Martín Schere
+    - Ola Ahlman <ola.ahlman@tastenkunst.io>
+Implementors: 
+    - JPG Store
+    - Eternl
+    - Typhon
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/443
+Created: 2023-09-03
+License: CC-BY-4.0
+---
+
+## Abstract
+This CIP extends [CIP-30 (Cardano dApp-Wallet Web Bridge)](https://cips.cardano.org/cips/cip30/) to provide an additional endpoint for dApp to sign multiple transactions in bulk.
+
+## Motivation: why is this CIP necessary?
+Currently, there is no way to sign multiple transactions in bulk, and the experience of signing a chain of transactions is suboptimal. We propose the addition of a signTxs endpoint that enable wallets to create an array of interconnected transactions and sign them all at once.
+
+## Specification
+
+### TransactionSignatureRequest
+
+```
+type = TransactionSignatureRequest {|
+  cbor: cbor\<transaction>,
+  partialSign: bool = false,
+|};
+```
+
+Used to represent a single transaction awaiting a user's signature. More details on {partialSign} can be found in [api.signTx](https://cips.cardano.org/cips/cip30/#apisigntxtxcbortransactionpartialsignboolfalsepromisecbortransactionwitnessset) defined in CIP-30.
+
+### api.cip????.signTxs(txs: TransactionSignatureRequest[]): Promise\<cbor\<transaction_witness_set>[]>
+
+Errors: `APIError`, `TxSignError`
+
+Signs a chain of transactions, which can be described as a sequence of interconnected transactions where each subsequent transaction depends on the previous one. The returned array values of the witness set directly correspond to the elements in the `txs` parameter, aligning the witness set at index 0 with the transaction at index 0, and so forth.
+
+## Rationale: how does this CIP achieve its goals?
+Allowing for bulk signing of transactions can greatly improve user experience in certain situations. There are however certain things that should be considered by wallets implementing this CIP, namely user visibility of what is signed. Though not explicitly specified in this CIP, as it would be up to the wallet to find a good solution, the wallet should make it clear to the user that multiple transactions are to be signed, and to give a clear overview of what is signed. In addition to visibility, the wallet shall process the input transaction array in order to allow transactions to be chained by accepting a previous transaction in the array to be used as input in a following transaction.
```

Comment:
> There are however certain things that should be considered by wallets implementing this CIP, namely user visibility of what is signed. Though not explicitly specified in this CIP, as it would be up to the wallet to find a good solution, the wallet should make it clear to the user that multiple transactions are to be signed, and to give a clear overview of what is signed.

This should be moved to the `Specification` section.

> shall process the input transaction array in order to allow transactions to be chained

This does not seem very well defined, Im not sure how this should work - are wallets allowed to reorder the inputted transactions? if so how can dApps track which signatures correspond to what tx.

Reply:
Ok, will move the mentioned section to specification. 

I guess bad wording on my part. Transactions MUST be processed in the order provided as input. This is what I meant by `in order` but clearly isn't well worded if it causes confusion. I will find a way to rewrite it to make it clear.

I will also see if I can extend the rational section to put a bit more meat on it.